gym/wrappers/rescale_action.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+import gym
+from gym import spaces
+
+
+class RescaleAction(gym.ActionWrapper):
+    r"""Rescale the continuous action space of the environment from a range of [a, b]. """
+    def __init__(self, env, a, b):
+        assert isinstance(env.action_space, spaces.Box), (
+            "expected Box action space, got {}".format(type(env.action_space)))
+        assert np.less_equal(a, b).all(), (a, b)
+        super(RescaleAction, self).__init__(env)
+        self.a = np.zeros(env.action_space.shape, dtype=env.action_space.dtype) + a
+        self.b = np.zeros(env.action_space.shape, dtype=env.action_space.dtype) + b
```

Comment:
I think the action space has to be modified, otherwise, in the wrapped environment expressions like `env.step(env.action_space.sample())` may cause an error.

Reply:
Thanks @pzhokhov , now the change of action space is added.